generator/tests/builtin-extensions/builtin-extensions

```diff
@@ -0,0 +1,52 @@
+@theia/vscode-builtin-bat@0.2.1
+@theia/vscode-builtin-clojure@0.2.1
+@theia/vscode-builtin-coffeescript@0.2.1
+@theia/vscode-builtin-configuration-editing@0.2.1
+@theia/vscode-builtin-cpp@0.2.1
+@theia/vscode-builtin-csharp@0.2.1
+@theia/vscode-builtin-css@0.2.1
+@theia/vscode-builtin-debug-auto-launch@0.2.1
+@theia/vscode-builtin-docker@0.2.1
+@theia/vscode-builtin-emmet@0.2.1
+@theia/vscode-builtin-fsharp@0.2.1
+@theia/vscode-builtin-git@0.2.1
+@theia/vscode-builtin-git-ui@latest
+@theia/vscode-builtin-go@0.2.1
+@theia/vscode-builtin-groovy@0.2.1
+@theia/vscode-builtin-grunt@0.2.1
+@theia/vscode-builtin-gulp@0.2.1
+@theia/vscode-builtin-handlebars@0.2.1
+@theia/vscode-builtin-hlsl@0.2.1
+@theia/vscode-builtin-html@0.2.1
+@theia/vscode-builtin-ini@0.2.1
+@theia/vscode-builtin-jake@0.2.1
+@theia/vscode-builtin-java@0.2.1
+@theia/vscode-builtin-javascript@0.2.1
+@theia/vscode-builtin-json@0.2.1
+@theia/vscode-builtin-less@0.2.1
+@theia/vscode-builtin-log@0.2.1
+@theia/vscode-builtin-lua@0.2.1
+@theia/vscode-builtin-make@0.2.1
+@theia/vscode-builtin-markdown@0.2.1
+@theia/vscode-builtin-markdown-language-features@0.2.1
+@theia/vscode-builtin-merge-conflict@0.2.1
+@theia/vscode-builtin-npm@0.2.1
+@theia/vscode-builtin-objective-c@0.2.1
+@theia/vscode-builtin-perl@0.2.1
+@theia/vscode-builtin-powershell@0.2.1
+@theia/vscode-builtin-pug@0.2.1
+@theia/vscode-builtin-python@0.2.1
+@theia/vscode-builtin-r@0.2.1
+@theia/vscode-builtin-razor@0.2.1
+@theia/vscode-builtin-ruby@0.2.1
+@theia/vscode-builtin-rust@0.2.1
+@theia/vscode-builtin-scss@0.2.1
+@theia/vscode-builtin-search-result@next
```

Comment:
next ?

Reply:
I will remove it until stable version appear.